.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Use # to add comments, use this example https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/717a11716169877e84c5ab672f1a93b3156ed3b9/.github/workflows/CodeQL%20Security%20Scan.yml#L1